software/openvisualizer/bin/openVisualizerApp/openVisualizerApp.py

```diff
@@ -107,6 +117,44 @@ def __init__(self,confdir,datadir,logdir,simulatorMode,numMotes,trace,debug,simT
                     moteHandler.hwSupply.INTR_SWITCHON
                 )
             self.simengine.resume()
+
+       
+        # import the topology from the json file
+        if self.pathTopo != '.' and self.simulatorMode == True and os.path.exists(pathTopo):
+            
+            # delete each connections automatically established during motes creation
```

Comment:
could you create an issue to avoid removing all connections and creating one. This is not a high priority at this point.